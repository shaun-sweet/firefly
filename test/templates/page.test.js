import Vue from 'vue'
import { shallow } from 'vue-test-utils'

import Page from '../../templates/page.vue'


describe('Page template', () => {
  it('should render the default page template', () => {
    const wrapper = shallow(Page)
    expect(wrapper.element).toMatchSnapshot();
  })
})
