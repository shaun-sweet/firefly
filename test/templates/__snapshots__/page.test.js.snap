// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page template should render the default page template 1`] = `
<div
  class="layout-padding"
/>
`;
